test/remote/gateways/remote_aafes_test.rb

```diff
@@ -0,0 +1,166 @@
+require 'test_helper'
+
+class RemoteAafesTest < Test::Unit::TestCase
+  def setup
+    @gateway = AafesGateway.new(fixtures(:aafes))
+    @gateway.ssl_strict = false # Fails to send request to gateway otherwise :shrug
```

Comment:
😬

Reply:
I took this out, not sure what was going on when I initially tested this.